blocks/api/paste/index.js

```diff
@@ -19,7 +19,7 @@ import msListConverter from './ms-list-converter';
 import listMerger from './list-merger';
 import imageCorrector from './image-corrector';
 import blockquoteNormaliser from './blockquote-normaliser';
-import { deepFilter, isInvalidInline, isNotWhitelisted, isPlain } from './utils';
+import { deepFilter, isInvalidInline, isNotWhitelisted, isPlain, isInline } from './utils';
```

Comment:
It looks like we don't care about the order of import statements. In Calypso they introduced codemod which sorts everything for you. Just saying, no action necessary here :)

Reply:
Yeah, I saw your ticket #2819. :)